src/language/statics/Coverage.re

```diff
@@ -509,60 +1213,217 @@ module Submatrices = {
 type result = {
   is_exhaustive: bool,
   redundant_rows,
+  unseen_pattern: UnseenPatternList.t,
```

Comment:
turn is_exhaustive field into a sum type with the unseen patterns only generated when it is inexhaustive